### DISCUSSION TITLE
[HTTP]:500 - [CorrelationId]:63fb53a1-5029-9000-f3d1-f9edba8b5dc0 [Version]:16.0.0.25304

### DISCUSSION THREAD

**Discussion by Wernbe** [microsoft-search/pnp-modern-search#4017] [Q&A]
**Version used**
4.13.1

**Describe the bug**
When using the search webparts PnP search filters and PnP - Search results to search for listitems with a certain content type and show them. We get sometimes in red on top of the PnP search results web part the error [HTTP]:500 - [CorrelationId]:63fb53a1-5029-9000-f3d1-f9edba8b5dc0 [Version]:16.0.0.25304. After refreshing a couple of times it disappears. 

**To Reproduce**
It comes when using the filters but sometimes also without using the filters.

**Screenshots**
![image](https://github.com/user-attachments/assets/9a25992c-3dd9-46b3-af73-b2faa5f36676)

**Desktop (please complete the following information):**
 - Edge

**Additional context**

**Comment by kasperbolarsen** [comment]
Can you verify if you get the same error when calling the API using one of the tools mentioned here, https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/
